src/components/survey/Survey.tsx

```diff
@@ -0,0 +1,90 @@
+"use client"
+import { ChangeEvent, useState, useCallback } from 'react';
+
+
+interface SurveyProps {
+  title: string;
+  lowest: string;
+  highest: string;
+}
+
+export const Survey = ({ title, lowest, highest  }: SurveyProps) => {
```

Comment:
Maybe I'm getting ahead of myself way too much right now, but I would like to consider this `Survey` component as a 'range survey item' rather than a 'regular' survey tool. What I envision is the following:

- Pass in some props: title, description, min and max (both objects with a numeric value and a description) and click/submit event handlers;
- Passing in the event handlers will allow us to reuse this component in other places (for example, if we were to create some type of elaborate form) with specific implementations of handling the event;
- Map the min and max numeric values into a range, which can be used for the amount of buttons to be rendered as part of this component; 

I would be very happy to elaborate on my vision for this, if you or other agree as well. 😄